doc/whats_new/v1.0.rst

```diff
@@ -105,10 +105,20 @@ Changelog
 :mod:`sklearn.inspection`
 .........................
 
-- |Fix| Allow multiple scorers input to
+- |Enhancement| Allow multiple scorers input to
   :func:`~sklearn.inspection.permutation_importance`.
   :pr:`19411` by :user:`Simona Maggio <simonamaggio>`.
 
+- |Enhancement| in :func:`~sklearn.inspection.plot_partial_dependence`, allow
+  `kind` to accept a list of strings to specify for each feature interaction
+  which type of plot a user wants to draw.
```

Comment:
would this be clearer? (the current one is fine too anyway)

```suggestion
  `kind` to be specified on a per-plot basis
```

Reply:
I find the current one more informative (even if a tad complex).